packages/react/src/analytics/integrations/GTM/eventsMapper.js

```diff
@@ -72,79 +337,144 @@ const eventsMapper = {
       product: getProductData(properties),
     };
   },
-  [eventTypes.CHECKOUT_STARTED]: data => {
+  [eventTypes.PLACE_ORDER_FAILED]: data => {
     const properties = getEventProperties(data);
 
     return {
       products: get(properties, 'products', []).map(getProductData),
-      orderId: properties.orderId,
-      total: properties.total,
```

Comment:
I think this property `total` could be missing.
I understand that was "replaced" by value, but for full compatibility, I think we need to guarantee that `total` stay on payload, so I suggest on new method, add `total` value, mapping to new `value` property.

Note: this could be relevant if some tenat had previous third party integration that use this specific event, and this specific value.

Reply:
Fixed! I've added a verification on `getEventTotalValue` method to check if total exists and, if so, map that to `value` property.